### DISCUSSION TITLE
Apologies ahead of time

### DISCUSSION THREAD

**Discussion by joebroke** [adrianlungu/ScreenSleep#14] [General]
I had this running on my last install of windows, but I don't know what I'm doing, or not doing, but I don't see an obvious executable to run.  Everything I try to open or run MS Edge hijacks.  I feel dumb but what am I doing wrong?

**Comment by adrianlungu** [comment]
Hey @joebroke 

First off, what did you download ? Was it the source code or the archive with the executable ?

Releases can be found here: https://github.com/AdrianLungu/ScreenSleep/releases

The `.7z` archive contains the compiled executable and config file.

**Reply by joebroke** [reply]
Oy! I'm a dope. Sorry for wasting your time.  and Thanks for the very useful program!
